### ISSUE TITLE
Focus Indicator Does Not Pass Accessibility 3:1 Ratio (1.4.11 Non-text Contrast (AA))

### ISSUE THREAD

**Issue by ericcastro916** [Office-of-Digital-Services/California-State-Web-Template-Website#1744]
Focus Indicator Does Not Pass 3:1 Ratio Accessibility Color Contrast Standards. It is currently at 2.7:1 which fails 1.4.11 Non-text Contrast (AA)
